lib/DDG/Spice/LaunchLibrary.pm

```diff
@@ -0,0 +1,31 @@
+package DDG::Spice::LaunchLibrary;
+# ABSTRACT: List upcoming rocket launches and where to watch them online
+
+use strict;
+use DDG::Spice;
+
+spice is_cached => 1;
+
+name "Launch Library";
+source "LaunchLibrary.net";
+description "List upcoming rocket launches and where to watch them online";
+primary_example_queries "upcoming rocket launches", "next space launch";
+category "reference";
+topics "geek", "science";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/LaunchLibrary.pm";
+attribution github => ["globalspin", "Chris Radcliff"],
+            twitter => "chris_radcliff";
+
+# Version 1.1 of the API allows GET requests. 
+# Parameters are included in the URL path; "/next/5" shows the next 5 launches
+# See launchlibrary.net for the API reference.
+spice to => 'http://launchlibrary.net/1.1/launch/next/5';
+spice wrap_jsonp_callback => 1;
+
+triggers any => "rocket launches", "rocket launch", "space launches", "space launch";
+
+handle remainder => sub {
+    return 1;
```

Comment:
A regex is a good idea. I was wondering how to handle variations on "upcoming rocket launch", "next space launch", "watch a rocket launch online", and "rocket launch schedule" without listing them all. 

I'll craft a regex and add some of these false-positives to the unit test.